src/lib/soup-to-svg.ts

```diff
@@ -147,12 +147,57 @@ function createSchematicComponent(
 ): any {
   const transform = `translate(${center.x}, ${center.y}) rotate(${(rotation * 180) / Math.PI})`;
 
-  if(symbolName) {
-    // TODO: Change the types in soup from string
-    return parseSync(getSvg((symbols as any)[symbolName], {
+  if (symbolName) {
+    const svg = parseSync(getSvg((symbols as any)[symbolName], {
       width: size.width,
       height: size.height,
-    }))
+    }));
+
+    // Filter out non-path elements and modify path colors
+    const pathElements = svg.children
+    .filter((child: any) => child.name === 'path' && child.attributes.fill !== 'green')
+    .map((path: any) => {
+      const currentStrokeWidth = Number.parseFloat(path.attributes['stroke-width'] || '0.02');
+      const newStrokeWidth = (currentStrokeWidth * 1.5).toString();
+      
+      return {
+        ...path,
+        attributes: {
+          ...path.attributes,
+          stroke: path.attributes.stroke === 'black' ? 'red' : path.attributes.stroke,
+          'stroke-width': newStrokeWidth
+        }
+      };
+    });
```

Comment:
can you run biome on these files, we should add the format workflow but i'm noticing future reviews are getting messed up whenever someone isn't biome'ing with the latest config

I think this approach is too complicated because the code path is really long, in your approach you're doing:

```
symbol data -> svg -> parse svg into json -> modify svg json -> stringify svg
```

can we just implement:

```
symbol data -> svg
```

I know there's a bit of code overlap between the two repos, but we're just parsing `SchSymbol.primitives.filter(p => p.type === "path")

I think this will make transform issues a lot less confusing as well, you won't need to do logic to figure out if a path is green etc.

Reply:
Yeah, this makes sense. I will take a look at this approach